openml/evaluations/functions.py

```diff
@@ -0,0 +1,93 @@
+import xmltodict
+
+from .._api_calls import _perform_api_call
+from ..evaluations import OpenMLEvaluation
+
+def list_evaluations(function, offset=None, size=None, id=None, task=None, setup=None,
+              flow=None, uploader=None, tag=None):
+    """List all run-evaluation pairs matching all of the given filters.
+
+        Perform API call `/evaluation/function{function}/{filters} 
```

Comment:
Shouldn't this be:

    ``/evaluation/function/{function}/{filters}``

?

Also, it would be great to add to the docstring a note where the user can find a list of _functions_.